README.md

```diff
@@ -97,17 +96,16 @@ Run `cargo install --path lanterndb_create_index` to install the binary
 
 Run `ldb-create-index --help` to show the cli options.
 
-```
+```bash
 Usage: ldb-create-index --uri <URI> --table <TABLE> --column <COLUMN> -m <M> --efc <EFC> --ef <EF> -d <DIMS> --metric-kind <METRIC_KIND> --out <OUT>
 ```
 
 ### Example
 
-```
+```bash
 ldb-create-index -u "postgresql://localhost/test" -t "small_world" -c "vec" -m 16 --ef 64 --efc 128 -d 3 --metric-kind cos --out /tmp/index.usearch
 ```
 
 ### Notes
 The index should be created from the same database on which it will be loaded, so row tids will match later.  
-Currently version of usearch is not up to date, so it will match with LanternDB's usearch version, but this version has bugs when creating index with more than 8k items.
-
+The current version of usearch is not up to date, so it will match with LanternDB's usearch version, but this version has bugs when creating index with more than 8k items.
```

Comment:
Thanks @dqii, can you also remove this part? This is not actual anymore as I have updated the version of usearch.

Reply:
Made this change!